docs/src/devdocs/interpolations.md

```diff
@@ -19,13 +19,12 @@ Ferrite.derivative(::Interpolation{dim}, ::Vec{dim}) where {dim}
 
 ```@docs
 Ferrite.value(::Interpolation, ::Int, ::Vec)
-Ferrite.vertices(::Interpolation)
-Ferrite.nvertexdofs(::Interpolation)
-Ferrite.faces(::Interpolation)
-Ferrite.nfacedofs(::Interpolation)
-Ferrite.edges(::Interpolation)
-Ferrite.nedgedofs(::Interpolation)
-Ferrite.ncelldofs(::Interpolation)
+Ferrite.vertexdof_indices(::Interpolation)
+Ferrite.edgedof_indices(::Interpolation)
+Ferrite.edgedof_interior_indices(::Interpolation)
+Ferrite.facedof_indices(::Interpolation)
+Ferrite.facedof_interior_indices(::Interpolation)
+Ferrite.celldof_interior_indices(::Interpolation)
```

Comment:
I dont see the need for "cell" "local" or "dof"...seems excessive?
Why not just `faceindices`, `edgeindices`, `edgeindices_interior` etc..

Reply:
`faceindices` is ambiguous with `FaceIndex`, hence I included dof. `celllocal` is ment to indicate that it is an index into the dof range of a single cell, in contrast to the dof index in e.g. the "global" dof range of the equation system (be it on a single process or in distributed to the index in the distributed sparse matrix range.

Reply:
But the `faceindices(::Interpolation)` would be "operating" on a `::Interpolation`, so I dont see why anyone would think they would get the global dofs. Btw, doesn't "dof" and "indices" describe the same thing here :P ?

I kind of get your first point though.

Reply:
I always assume that (new) users don't have very technical knowledge about internals of a package. 

For me a dof is also an index represented as an index, but here I want to refer to a (local) index, which you need to obtain the dof (index) locally on a cell. So the dof index is the index into the solution vector and system matrix, while this index is the index locally in a cell, to get the dof index. Does that make sense?

Reply:
Ok i see your point. But I still think your suggestion is a bit long and clunky. I guess it does not matter to much since it a internal method. But lets see if the other ferrite devs have any suggestions/opinions.